ReactiveTask/Task.swift

```diff
@@ -175,6 +175,8 @@ private final class Pipe {
 					}
 				}, completed: {
 					dispatch_io_close(channel, 0)
+				}, interrupted: {
```

Comment:
Sending Interrupted event disposes the disposable I think, so `dispatch_io_close(channel, DISPATCH_IO_STOP)` would be called finally?